Cassiopee/KCore/Dist.py

```diff
@@ -83,6 +83,7 @@ def getenv(name):
 #==============================================================================
 def getDataFolderName(name='Data'):
     elsaprod = os.getenv("ELSAPROD")
+    if elsaprod=='junoMain': elsaprod='juno'
```

Comment:
Why do you need this prod to be listed here? It is identical to juno from what I can see

Reply:
It is for the validCassiopee global database. Data_juno exists but Data_junoMain doesn't. As such, it will try to create it. However, I just want it to take the data that is in Data_juno. Hence, the local variable change elsaprod=juno.